src/nodes/geometry/RangeNode.js

```diff
@@ -65,11 +106,11 @@ class RangeNode extends Node {
 			max.setScalar( 0 );
 
 			if ( minLength === 1 ) min.setScalar( minValue );
-			else if ( minValue.isColor ) min.set( minValue.r, minValue.g, minValue.b );
+			else if ( minValue.isColor ) min.set( minValue.r, minValue.g, minValue.b, 1 );
```

Comment:
@sunag This is minor fix to prevent `NaN` values in the attribute. `Vector4.set()` should always be used with four arguments otherwise properties end up `undefined` and then produce garbage values in subsequent computations.